redesign.user.js

```diff
@@ -4866,6 +5090,25 @@ ready(() => {
     document
         .querySelector('#usernavigation .usermenu-container')
         ?.before(settingsBtnWrapper);
+    if (
+        (unseenSettings.size &&
+            getSetting('general.highlightNewSettings.navbar')) ||
+        IS_NEW_INSTALLATION
```

Comment:
`IS_NEW_INSTALLATION` is only true on the very first visit of any moodle page after installing the script (which includes the login page). 

The login page does not include the settings button. Therefore a new user might not see the "new"-label, when not logged in on their first visit. 

Maybe store when Settings are first visited (?)

Reply:
fixed in 7af15fd3.